ccxt.js

```diff
@@ -20732,12 +20849,7 @@ var yobit = extend (btce, {
                     let lowercase = currencies[i];
                     let uppercase = lowercase.toUpperCase ();
                     let currency = this.commonCurrencyCode (uppercase);
-                    let account = undefined;
-                    if (currency in result) {
-                        account = result[currency];
-                    } else {
-                        account = this.account ();
-                    }
+                    let account = this.extend (this.account (), result[currency]);
```

Comment:
Ah, unfortunately it fails on Python, because result[currency] is illegal in Python if `result` does not have such key... But don't bother, I'll fix for that, it just a Python-only issue and you didn't have to know that. No worries.